packages/@core/base/design/src/css/global.css

```
@@ -54,24 +54,146 @@ a:visited {
-::view-transition-new(root) {
+::view-transition-new(theme) {
   @apply z-[2147483646];
 }
 
-html.dark::view-transition-old(root) {
+html.dark::view-transition-old(theme) {
   @apply z-[2147483646];
 }
 
-html.dark::view-transition-new(root) {
+html.dark::view-transition-new(theme) {
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)